### DISCUSSION TITLE
Offloading data to cloud storage for RabbitMQ Streams

### DISCUSSION THREAD

**Discussion by glothriel** [rabbitmq/rabbitmq-server#6766] [Ideas] [answered]
Hi. I'm wondering if there are any plans for offloading streams data (for example basing on some policy, like tiers with TTL) to object storage to reduce disk costs? It would be a really great feature.

Similar feature in redpanda: https://docs.redpanda.com/docs/platform/data-management/tiered-storage/

**Comment by michaelklishin** [comment, marked as answer]
There are no plans for such a feature at the moment. RabbitMQ streams usually do not retain their data for weeks or months, so there is no demand for this feature from our user base.

**Reply by glothriel** [reply]
Is storing large amount of data or just old data considered a bad practice when using streams?
